config/locales/flash.en.yml

```diff
@@ -1,6 +1,7 @@
 en:
   form:
     required: "Fill all required fields"
+    forbidden: "Unauthorised access to Edit"
```

Comment:
Can you change the message?

forbidden: You don't have permission to edit

Reply:
Ok @iMagesh